ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -483,7 +495,7 @@ def update(self, batch: AgentBuffer, num_sequences: int) -> Dict[str, float]:
         offset = 1 if self.policy.sequence_length > 1 else 0
         next_memories_list = [
             ModelUtils.list_to_tensor(
-                batch[BufferKey.MEMORY][i][self.policy.m_size // 2 :]
```

Comment:
Are there changes to memories in this PR? Why was this line changed?

Reply:
https://github.com/Unity-Technologies/ml-agents/pull/4948